### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by rvl** [multani/sonata#117]
Hi, I maintain the sonata package in nixpkgs. I have found that version 1.7b1 is failing to run due to this error:
```
Traceback (most recent call last):
  File "/nix/store/g082g5mkck9qfnlfqzic1qz95zag70md-sonata-1.7b1/lib/python3.8/site-packages/sonata/launcher.py", line 197, in on_application_activate
    sonata = main.Base(args)
  File "/nix/store/g082g5mkck9qfnlfqzic1qz95zag70md-sonata-1.7b1/lib/python3.8/site-packages/sonata/main.py", line 759, in __init__
    self.iterate_now()
  File "/nix/store/g082g5mkck9qfnlfqzic1qz95zag70md-sonata-1.7b1/lib/python3.8/site-packages/sonata/main.py", line 1064, in iterate_now
    self.iterate()
  File "/nix/store/g082g5mkck9qfnlfqzic1qz95zag70md-sonata-1.7b1/lib/python3.8/site-packages/sonata/main.py", line 1025, in iterate
    self.handle_change_status()
  File "/nix/store/g082g5mkck9qfnlfqzic1qz95zag70md-sonata-1.7b1/lib/python3.8/site-packages/sonata/main.py", line 1399, in handle_change_status
    self.volumebutton.set_value(int(self.status['volume']))
KeyError: 'volume'
```
However, the latest commit from git (d6445e21d20f66edd1d2d53d329e654b4dce3cdc) works.
The nixpkgs project generally prefers to make packages from release tags rather than git commits.
Is it possible to tag a new release for this bug fix?

**Comment by solbu** [comment]
> Is it possible to tag a new release for this bug fix?

I support this request. I am the Mageia maintainer of sonata, and I have been waiting for the same thing.

> The nixpkgs project generally prefers to make packages from release tags rather than git commits.

Most distros do this, they prefer releases over commit IDs.
Some even refuses to use a newer commit if an older Release exist. Then they wait for a new release, and might implement some commits as patches to the latest release.

**Comment by multani** [comment]
I'll create a new tag this week :+1:

**Comment by multani** [comment]
I released [v1.7.0](https://github.com/multani/sonata/releases/tag/v1.7.0) :tada:

**Comment by rvl** [comment]
Thanks @multani - the Sonata package in nixpkgs unstable is now updated: NixOS/nixpkgs#107605.
